config/routes.rb

```diff
@@ -1,9 +1,13 @@
 Rails.application.routes.draw do
+  root "photos#index"
+
+  resources :likes
+  resources :follow_requests
   resources :comments
   resources :photos
   devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   root "photos#index"
```

Comment:
Please remove duplicate root route. You are currently defining the root route twice